src/main/java/com/jenkinsci/plugins/badge/action/BadgeSummaryAction.java

```diff
@@ -24,8 +24,8 @@
 package com.jenkinsci.plugins.badge.action;
 
 import java.io.Serial;
-import org.apache.commons.lang.StringEscapeUtils;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
Note that this is deprecated:

> Deprecated.
> As of 3.6, use Apache Commons Text [StringEscapeUtils](https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html) instead

Reply:
Ah, deprecation notice did not pop up in my IDE. Will update in a separate PR. Thanks for letting me know!